Alcancia-io/alcancia
# My account screen

## Summary
Moved delete account button to empty (for now) Account screen.
Also changed router redirect behavior to avoid adding empty conditions for every new screen

## Requirements
None

## Type of change

## How to test?
Log in, go to profile screen and tap on my account. Logging out and deleting your account should take you to the welcome screen.

## How has this been tested?
- [ ] Tapping on my account button takes you to new screen without routing errors in the console
- [ ] Hot restarting app redirects to correct screen

## Checklist (for the reviewer)
